packages/block-editor/src/components/block-list/block-invalid-warning.js

```diff
@@ -59,21 +46,33 @@ export default function BlockInvalidWarning( { clientId } ) {
 	const [ compare, setCompare ] = useState( false );
 	const onCompareClose = useCallback( () => setCompare( false ), [] );
 
-	const attemptBlockRecovery = useCallback( () => {
-		replaceBlock( block.clientId, recoverBlock( block ) );
-	}, [ block, replaceBlock ] );
-
-	const convertToClassic = useCallback( () => {
-		replaceBlock( block.clientId, blockToClassic( block ) );
-	}, [ block, replaceBlock ] );
-
-	const convertToHTML = useCallback(
-		() => replaceBlock( block.clientId, blockToHTML( block ) ),
-		[ block, replaceBlock ]
-	);
-
-	const convertToBlocks = useCallback(
-		() => replaceBlock( block.clientId, blockToBlocks( block ) ),
+	const convert = useMemo(
```

Comment:
I could destructure it, but I decided to keep it this way as it allows me to pass a single `convert` into the following `useMemo` dependency array..